### PR TITLE
Generator for OpenSSL known logs list

### PR DESCRIPTION
**Pull request by RJPercival**

### CODE REVIEW

**Comment by eranmes** [comment]
Is the openssl format documented anywhere?

**Comment by RJPercival** [comment]
It is documented here: https://github.com/openssl/openssl/blob/master/doc/ssl/SSL_CTX_set_ctlog_list_file.pod

The additional fields output by this generator do no harm, and may be useful to users of OpenSSL, so I've included them too.

**Comment by RJPercival** [comment]
Looks like there's a flaky test: 
```
cpp/log/cluster_state_controller_test.cc:386: Failure
Value of: sth.ValueOrDie().tree_size()
Actual: 1000
Expected: cns2.newest_sth().tree_size()
Which is: 1001
[  FAILED  ] ClusterStateControllerTest.TestUsesLargestSTHWithIdenticalTimestamp (2024 ms)
```

File: python/utilities/log_list/openssl_generator.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+def _enabled_logs_conf(logs):
```
Comment (eranmes) [inline python/utilities/log_list/openssl_generator.py]: Add a link to the format documentation.

File: python/utilities/log_list/openssl_generator.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    return "enabled_logs = %s\n" % (", ".join(log["url"] for log in logs))
+
+def _log_conf(log, operators):
+    return (
+        "[%(name)s]\n"
+        "description = %(description)s\n"
+        "key = %(key)s\n"
+        "url = https://%(url)s\n"
+        "operated_by = %(operated_by)s\n" % {
+            "name": log["url"],
```
Comment (eranmes) [inline python/utilities/log_list/openssl_generator.py]: Did you intend to use the url as name?

**Comment by eranmes** [comment]
Would LGTM once comments are addressed and you try travis one more time.

File: python/utilities/log_list/openssl_generator.py (hunk 1 above)
Comment (RJPercival) [inline python/utilities/log_list/openssl_generator.py]: Done.

File: python/utilities/log_list/openssl_generator.py (hunk 2 above)
Comment (RJPercival) [inline python/utilities/log_list/openssl_generator.py]: Yes, the URL is guaranteed to be unique and is perhaps more readable than the log description when many are concatenated together for the "enabled_logs" line.

**Comment by RJPercival** [comment]
Different flaky test this time:
```
cpp/log/cluster_state_controller_test.cc:303: Failure
Value of: sth.ValueOrDie().timestamp()
Actual: 100
Expected: newer_sth.timestamp()
Which is: 101
[  FAILED  ] ClusterStateControllerTest.TestKeepsNewerSTH (1005 ms)
```

**Comment by eranmes** [comment]
LGTM.

**Pull request merged**